Packs/LookoutMobileEndpointSecurity/Integrations/LookoutMobileEndpointSecurityEventCollector/LookoutMobileEndpointSecurityEventCollector.py

```diff
@@ -0,0 +1,249 @@
+import demistomock as demisto
+from CommonServerPython import *
+import urllib3
+from requests import Response
+
+from sseclient import SSEClient
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+VENDOR = 'lookout_mobile'
+PRODUCT = 'endpoint_security'
+BASE_URL = 'https://api.lookout.com/'
+FETCH_SLEEP = 10
+FETCH_INTERVAL_IN_SECONDS = 60
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any Demisto logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    """
+
+    def __init__(self, base_url: str, verify: bool, proxy: bool, event_type_query: str, app_key: str):
+        self.event_type_query = event_type_query
+        self.app_key = app_key
+        self.base_url = base_url
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+
+    def refresh_token_request(self) -> dict:
+        """Request a new access token"""
+        data = {'grant_type': 'client_credentials'}
+        headers = {'Authorization': f'Bearer {self.app_key}'}
+        response = self._http_request(method="POST", data=data, url_suffix='oauth2/token', headers=headers)
```

Comment:
Consider adding exception handling for the request (try-except) for cases of 401, 500

Reply:
Those cases are handled by the _http_request function